lib/routes/sohumobile/index.ts

```diff
@@ -0,0 +1,51 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/limit',
+    categories: ['traditional-media'],
+    example: '/sohumobile/limit',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['m.sohu.com/limit'],
+            target: '/limit',
+        },
+    ],
+    name: '手机搜狐网-新闻',
+    maintainers: ['asqwe1'],
+    handler: async () => {
+        const response = await ofetch(`https://m.sohu.com/limit`);
```

Comment:
```suggestion
        const response = await ofetch('https://m.sohu.com/limit');
```
Only use template literals when necessary.